pol-core/support/scripts/uo.em

```diff
@@ -274,7 +274,7 @@ CreateItemInContainer( container, objtype, amount := 1 );
 CreateItemInInventory( container, objtype, amount := 1 );
 CreateMenu( title );
 CreateMultiAtLocation( x, y, z, objtype, flags := 0, realm := _DEFAULT_REALM );
-CreateNpcFromTemplate( template, x, y, z, override_properties := 0, realm := _DEFAULT_REALM);
+CreateNpcFromTemplate( template, x, y, z, override_properties := 0, realm := _DEFAULT_REALM, force := 0);
```

Comment:
I would suggest to use a more verbose name for this parameter.
"force" could be anything:)
Maybe something like "forcelocation" that it has equal naming like the flag for MoveObject.

Reply:
Fixed. Sorry it took a while, been a busy couple of weeks.